### ISSUE TITLE
Regarding modeling issues

### ISSUE THREAD

**Issue by Moxing529** [cselab/aphros#25]
Hello, Dr. Karnakov!
Do you have any examples for polygon modeling using Polygon(self, origin, normal, right, normalrange, scale, polygon, **kwargs) and Ruled(self, origin, normal, right, normalrange, scale0, scale1, polygon0, polygon1, **kwargs)? I tried using them but was unable to model successfully.
Additionally, the current modeling method seems to only allow for the assembly of regular shapes. Could you suggest any methods for more complex modeling?
<img width="299" alt="6180d2880ecdf7c3ece7be0375a5261" src="https://github.com/user-attachments/assets/2e86cfa4-717c-4d27-b867-2dc840c6cf67" />
<img width="452" alt="ba0341cdc6ee353452d240209f2b317" src="https://github.com/user-attachments/assets/99c7bead-48cf-4aa7-a4a8-edc1baa73fe3" />

**Comment by Moxing529** [comment]
Does the sphere primitive or other similar operations in modeling have the capability to create a semicircle? Also, in 205/crystal, does vfoam refer to the average flow velocity of bubbles after entering the main channel? Thank you！

**Comment by pkarnakov** [comment]
Here's an example for using polygons (not sure if it's still working)
https://github.com/cselab/aphros/blob/master/examples/200_primlist/case/polygon

Also examples here (in particular "Tesla valve" and "Pipe")
https://github.com/cselab/aphros/wiki/Aphros-Explorer

> Does the sphere primitive or other similar operations in modeling have the capability to create a semicircle?

so to make a semicircle I'd create a circle ("sphere" with a large radius in the z direction) and then subtract a box.

> Also, in 205/crystal, does vfoam refer to the average flow velocity of bubbles after entering the main channel?

From `inletvel = num.vfoam * num.width / inletwidth`, I guess it is the average flow velocity, but it is only used for a velocity inlet. If `inletpressure_add` or `inletpressure` is specified (as done in all examples), that parameter is not used and will not affect anything.

**Comment by Moxing529** [comment]
But when I use a pressure inlet, changing vfoam still affects the size of the generated bubbles. Also, in \( n.Ca = p.mu\_liquid \times p.vfoam / p.sigma \), does vfoam also have some influence?

**Comment by pkarnakov** [comment]
Right, `vfoam` is used for computing `Ca` in `Numbers()`. Eventually the five numbers there should define the problem. So if `Ca` or `sidevel_ratio` changes, the size of bubbles may also change. I'm not sure I recall all the details now, but I'd base all calculations on the dimensionless parameters instead of physical units.
